targets/unity-v2/source/Shared/Internal/PlayFabHttp/PlayFabHTTP.cs

```diff
@@ -186,17 +186,39 @@ public static void SimplePostCall(string fullUrl, byte[] payload, Action<byte[]>
         /// </summary>
         protected internal static void MakeApiCall<TResult>(string apiEndpoint,
             PlayFabRequestCommon request, AuthType authType, Action<TResult> resultCallback,
-            Action<PlayFabError> errorCallback, object customData = null, Dictionary<string, string> extraHeaders = null, bool allowQueueing = false)
+            Action<PlayFabError> errorCallback, object customData = null, Dictionary<string, string> extraHeaders = null, bool allowQueueing = false, PlayFabAuthenticationContext authenticationContext = null, PlayFabApiSettings apiSettings = null)
             where TResult : PlayFabResultCommon
         {
             InitializeHttp();
             SendEvent(apiEndpoint, request, null, ApiProcessingEventType.Pre);
 
+            string developerSecretKey = null;
+            #if ENABLE_PLAYFABSERVER_API || ENABLE_PLAYFABADMIN_API
+                if(request.AuthenticationContext != null && !string.IsNullOrEmpty(request.AuthenticationContext.DeveloperSecretKey))
+                {
+                    developerSecretKey = request.AuthenticationContext.DeveloperSecretKey;
+                }
+                if(developerSecretKey == null && authenticationContext != null && !string.IsNullOrEmpty(authenticationContext.DeveloperSecretKey))
+                {
+                    developerSecretKey = authenticationContext.DeveloperSecretKey;
+                }
+                if(developerSecretKey == null)
+                {
+                    developerSecretKey = PlayFabSettings.DeveloperSecretKey;
+                }
+            #else
+                developerSecretKey = PlayFabSettings.DeveloperSecretKey;
```

Comment:
Please remove these two lines. If server or admin APIs are not enabled, we should not even be using developer secret keys